resource_retriever/CMakeLists.txt

```diff
@@ -59,4 +94,9 @@ install(
   DESTINATION include
 )
 
+install(
+  DIRECTORY test
+  DESTINATION share/${PROJECT_NAME}
+)
```

Comment:
Why are we installing the tests?  I don't think this is something we want in the "released" binaries.

Reply:
The test is looking for a specific file https://github.com/ros/resource_retriever/blob/a0b44a592dc55408f5515a07900a39eced41724d/resource_retriever/test/test.cpp#L40

I think this should be installed otherwise the code will not find it

Reply:
While that is true, it also means that the tests will all make it into a binary once we build it.  I don't think that is desirable.

I'm not quite sure how to do this and still test that this is looking things up in the package share directory.  Any other thoughts on how to do this?  At the very least, we should only install `test.txt`, not all of the `test` subdirectory.

Reply:
https://github.com/ros/resource_retriever/pull/43/commits/63fc6e7cca798dd894508c6dbc031f25138a55a8

Reply:
Thanks, that's a bit better.  @sloretz Any other thoughts here?